### ISSUE TITLE
Create shortcut in %AppData% root

### ISSUE THREAD

**Issue by GeertvanHorrik** [Squirrel/Squirrel.Windows#195]
I had an interesting question. How can I run the app (latest version) without known the path? I think that's a good question and can be solved by creating a shortcut in the app data root that is independent of the version. This way someone that wants to automate calls can that shortcut to run the app.

For example:

%LocalAppData%\app-1.1.0-unstable0028\app.exe

then changes to 

%LocalAppData%\app-1.1.0-unstable0031\app.exe

If app.exe (shortcut) would exist beside Update.exe, the user can simply run that.

**Comment by flagbug** [comment]
I had this exact problem when I wanted to create a portable application with Squirrel, so I've sent PR #155 some time ago

`ShortcutLocation.AppRoot` is a thing, but you have to manually set it via the Squirrel events I think

**Comment by GeertvanHorrik** [comment]
Thanks, awesome! Then I vote for making it default :-)

**Comment by paulcbetts** [comment]
There's a feature in Update.exe built-in to do exactly this:

```bash
update.exe --processStart app.exe    ## Always starts the latest version of app.exe
```

**Comment by GeertvanHorrik** [comment]
Cool, didn't know. Really nice feature, thanks for letting me know.

**Comment by paulcbetts** [comment]
Oh wait, belay that, this won't work for you - I need an extra flag. `--processStart` was an internal flag originally designed to implement `RestartApp` and as a result, it waits for its parent to exit before running the app you give it. I need to add an extra `--dontWait` flag before this is Generally Useful

**Comment by kevfromireland** [comment]
I have had users report the same issue. They pin the squirrel created shortcut to their taskbar and then during update their shortcut points to a broken location, or worse, points to an old build.

The actual path of the current version should probably be hidden as an implementation detail and any shortcuts created using Squirrel point to something in the app root which redirects to use the latest installed version?

**Comment by paulcbetts** [comment]
@kevfromireland I don't think these are the same issues, this is a feature request. We attempt to fix up pinned shortcuts but maybe that isn't happening anymore

**Comment by GeertvanHorrik** [comment]
I have written a custom tool that detects the latest installed version in both program files and local app data (we support both) to run an app, so this issue can be closed.

**Comment by paulcbetts** [comment]
> The actual path of the current version should probably be hidden as an implementation detail and any shortcuts created using Squirrel point to something in the app root which redirects to use the latest installed version?

I actually implemented this in 0.9.0, but unfortunately it doesn't help - when you pin an app, Explorer completely ignores any existing shortcut and creates a private one, directly to your EXE

**Comment by SissiKonsta** [comment]
Is there any update regarding this issue? I'm using squirrel.windows 1.4.4 and still has this issue. When the user pins the icon application its target is set to the running version and not to the Update.exe --processStart. How can we do this? Please, if anyone has resolved this issue, could they explain me how they did it?

Thanks
